components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/TrustedApp.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2024, WSO2 Inc. (http://www.wso2.com).
```

Comment:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```

check the license header